production/user_guide.md

```diff
@@ -125,142 +125,158 @@ You can repeat this process with the `/_version` and `/api/v1/metadata` endpoint
+The `bulkData` category provides information about beneficiaries. As shown below, there are three types of information (`Patient`, `Coverage`, and `Explanation of Benefit`) within the `bulkData` category that can be retrieved from two different endpoints (`/Group` and `/Patient`).
 
-The `bulkData` category provides information about beneficiaries.  As shown below, there are three pieces of information within the `bulkData` category.  The first one -- the `Patient` endpoint -- is an endpoint that returns information about your ACO’s assigned or assignable beneficiaries. The last two pieces of information -- jobId and filename -- return information about the request you’re making.
+<img class="ug-img" src="/assets/img/swagger_walkthrough_06.svg" alt="Swagger: Example of one of the two bulk data endpoints and the available resource types">
 
-Within the `Patient` endpoint, you can make requests for up to three resource types:
+Within either endpoint, you can make requests for up to three resource types:
+1. **ExplanationOfBenefit** includes the bulk of claims data for your beneficiaries.
+2. **Patient** includes identification information about your assigned or assignable beneficiaries.
+3. **Coverage** data includes each beneficiary’s Medicare coverage plan.
 
-* **Explanation of Benefit** data includes claims data for your beneficiaries.
-* **Patient** data includes identification information about your assigned or assignable beneficiaries.
-* **Coverage** data includes each beneficiary’s Medicare coverage plan.
+There are two endpoints offered by BCDA that you may retrieve data from. They have similar capabilities, save for a few exceptions.
+1. **Patient Endpoint:** You may request all three resource types: `ExplanationOfBenefit`, `Patient`, and/or `Coverage` data for all beneficiaries. Filter data by date using the `_since` parameter.
+2. **Group Endpoint:** You may request `ExplanationOfBenefit`, `Patient`, and/or `Coverage` data for all beneficiaries. Filter data by date using the `_since` parameter. Calls to this endpoint must include a `Group` identifier. The only supported identifier at this time is “`all`”. The `all` identifier simply designates that you would like to pull claims data for all of your attributed beneficiaries.
 
-<img class="ug-img" src="/assets/img/swagger_walkthrough_06.svg" alt="Swagger: use the 'type' parameter to specify resource type(s) for your request" />
+**Note: The `/Group` endpoint differs from the `/Patient` endpoint in one key area. When filtering data using `_since` from the `/Group` endpoint, you will receive claims data since the date of your choice for existing beneficiaries AND you will also receive 7 years of historical data for all beneficiaries that are newly attributed to your ACO. See the [“Filtering your requests using `_since`”](https://bcda.cms.gov/production/user-guide/####-filtering-your-requests-using-_since){:target="_blank"} section for details and examples.**
```

Comment:
Link to "Filtering your requests" does not point to correct section -- needs a single, not four, hashtags.

Reply:
Fixed.